frontend/src/NavigationPath.tsx

```diff
@@ -151,12 +151,19 @@ export enum NavigationPath {
   createPolicyAutomation = '/multicloud/governance/policyautomation/create/:namespace/:name',
   editPolicyAutomation = '/multicloud/governance/policyautomation/edit/:namespace/:name',
   policyDetails = '/multicloud/governance/policies/details/:namespace/:name',
+  discoveredByCluster = '/multicloud/governance/discovered/:apiGroup?/:apiVersion/:kind/:policyName/:policyNamespace',
+  configByCluster = '/multicloud/governance/discovered/details/ConfigurationPolicy/:name/',
+  operatorByCluster = '/multicloud/governance/discovered/details/OperatorPolicy/:name/',
+  certByCluster = '/multicloud/governance/discovered/details/CertificatePolicy/:name/',
```

Comment:
These look to be unused after the refactor.
```suggestion
```

Reply:
Thanks!!